<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SF</class>
 <widget class="QWidget" name="SF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SF</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>what camera to open</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnOpenCamera">
           <property name="text">
            <string>OPEN CAMERA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCloseCamera">
           <property name="text">
            <string>CLOSE CAMERA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCaptura">
           <property name="text">
            <string>CAPTURA </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnWrite">
           <property name="text">
            <string>WRITE CSV</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
